src/components/TweetAuthor.js

```diff
@@ -1,22 +1,27 @@
 import React, { PropTypes } from 'react';
-import './TweetAuthor.css';
+import './style/TweetAuthor.css';
 
 const propTypes = {
-  user: PropTypes.object.isRequired
+  user: PropTypes.object.isRequired,
+  showModal: PropTypes.func.isRequired
```

Comment:
This would be better named `onClick` that way the TweetAuthor component could be reused in other places and the onClick behaviour changed. Calling it `showModal` ties it down to just doing that.